build.sbt

```diff
@@ -631,27 +632,27 @@ def customCommands: Seq[Setting[_]] = Seq(
    */
   commands += Command.command("release-sbt") { state =>
     // TODO - Any sort of validation
-    "so clean" ::
-      "allPrecompiled/clean" ::
-      "checkCredentials" ::
-      "conscript-configs" ::
+    "checkCredentials" ::
+    "clean" ::
+    "allPrecompiled/clean" ::
       "allPrecompiled/compile" ::
+      "allPrecompiled/publishSigned" ::
+      "so clean" ::
+      "conscript-configs" ::
       "so compile" ::
       "so publishSigned" ::
-      "allPrecompiled/publishSigned" ::
       "publishLauncher" ::
       state
   },
+  // stamp-version doesn't work with ++ or "so".
   commands += Command.command("release-nightly") { state =>
     "stamp-version" ::
-      "so clean" ::
+      "clean" ::
       "allPrecompiled/clean" ::
       "allPrecompiled/compile" ::
-      "so compile" ::
-      "so publish" ::
       "allPrecompiled/publish" ::
-      "publishLauncher" ::
+      "compile" ::
+      "publish" ::
```

Comment:
I don't think we were before either since Jenkins was issuing `clean stamp-version publish`.